android/app/src/main/java/com/chorestory/app/ParentLoginActivity.java

```diff
@@ -40,18 +61,35 @@ public void onClick(View v) {
                 username = usernameEditText.getText().toString();
                 password = passwordEditText.getText().toString();
 
-                // TODO: Authenticate user
+                LoginRequest loginRequest = new LoginRequest(username, password);
+                Call<LoginResponse> loginQuery = retrofitInterface.login(loginRequest);
+
+                loginQuery.enqueue(new Callback<LoginResponse>() {
+                    @Override
+                    public void onResponse(Call<LoginResponse> call, Response<LoginResponse> response) {
+                        if (response.isSuccessful()) {
```

Comment:
does `response` need a null check first?

Reply:
nope, response and response.body() are handled but I'll add a check for getToken()